addons/script-dock/plugin.cfg

```
@@ -1,7 +1,7 @@
 [plugin]
 
 name="script-dock"
-description=""
-author=""
+description="Moves the script list from ScriptEditor to a dock. This increases the available horizontal space for the ScriptEditor, and allows you to make the list a floating window."
+author="Koliur Rahman"
```

😅 I forget my real name is on github. I think just my username is ok
```suggestion
author="dugramen"
```